jme3-core/src/main/java/com/jme3/shadow/AbstractShadowRenderer.java

```diff
@@ -77,7 +79,7 @@
 public abstract class AbstractShadowRenderer implements SceneProcessor, Savable, JmeCloneable, Cloneable {
 
     protected static final Logger logger = Logger.getLogger(AbstractShadowRenderer.class.getName());
-
+    private static LightFilter NULL_LIGHT_FILTER = new NullLightFilter();

```

Comment:
Can you please add `final` modifier? (here and in the other classes you have used NullLightFilter?

Reply:
sure.

Reply:
thanks